src/tls_rsa.c

```diff
@@ -0,0 +1,1579 @@
+#include "tls_rsa.h"
+
+#if defined(MG_TLS) && MG_TLS == MG_TLS_BUILTIN
+
+#define NS_INTERNAL static
+typedef struct _bigint bigint; /**< An alias for _bigint */
```

Comment:
How I hate obvious comments... (and star comments, particularly in a single line...)